src/web/app/components/question-edit-form/question-edit-form.component.ts

```diff
@@ -192,9 +192,18 @@ export class QuestionEditFormComponent implements OnInit {
    * Change the {@code giverType} and {@code recipientType} and reset the visibility settings.
    */
   changeGiverRecipientType(giverType: FeedbackParticipantType, recipientType: FeedbackParticipantType): void {
+    // check if current recipientType is allowed for giverType,
+    // if not, set default recipientType to the first allowed type as default.
+    /* tslint:disable: no-non-null-assertion */
```

Comment:
The is the technique to disable tslint for an entire block. It is more appropriate to disable **the current line** here.